### PR TITLE
Nette presenter support

### PR DESCRIPTION
**Pull request by Lexinek**
Fixed errors for presenters descendant from `Nette\Application\UI\Presenter`

### CODE REVIEW

**Comment by Lexinek** [comment]
Fixes #7

File: app/presenters/NettePresenter.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?php declare (strict_types = 1);
+/**
+ * @author      Jan Mikes <mikes@v-i-c.eu>
+ * @copyright   Velveth International Corporation <v-i-c.eu>
+ */
```
Comment (TomasVotruba) [inline app/presenters/NettePresenter.php]: Please drop these, I try to keep code clean.

You'll be already mentioned in repository contributors. Would that be ok for you?

Comment (Lexinek) [inline app/presenters/NettePresenter.php]: Sure, i have it autogenerated and as well auto hidden by IDE :-). I will remove it.

Comment (TomasVotruba) [inline app/presenters/NettePresenter.php]: I see. I recommend dropping this spam :).

all commands => File Template => Include => File Header => delete all

**Comment by TomasVotruba** [comment]
Mergin!

**Pull request merged**